pkg/kafka/producer/kafka_test.go

```diff
@@ -47,7 +47,9 @@ func doListenerTLSTest(t *testing.T, serverTLSConfig *tls.Config, caCert, client
 	seedBroker := sarama.NewMockBrokerListener(t, 1, seedListener)
 	defer seedBroker.Close()
 
-	seedBroker.Returns(new(sarama.MetadataResponse))
+	metadataResponse := new(sarama.MetadataResponse)
+	metadataResponse.AddBroker(seedBroker.Addr(), seedBroker.BrokerID())
```

Comment:
Is this to make metadataResponse more accurate?

Reply:
No, that's required because of a sarama library change: https://github.com/IBM/sarama/commit/c42b2e0ed061fbdbade7fcb86ddf88a000a7bc4d

Reply:
understood